week3/predict_citibike.Rmd

```diff
@@ -0,0 +1,391 @@
+---
+title: "predict_citibike"
+author: "Sushobhan Parajuli"
+date: "`r Sys.Date()`"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(tidyverse)
+library(dplyr)
+library(scales)
+library(modelr)
+library(lubridate)
+```
+
+```{r}
+# import the holidays date
+# join holidays with the trips_per_day
+
+holidays <- read_csv("US Bank holidays", col_names = c("row", "ymd", "holiday"))

```

Comment:
You should include the data or cite where you got it if you're using outside data.

Also no spaces in filenames!!